projects/plugins/jetpack/modules/masterbar/nudges/bootstrap.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Bootstrap file for the nudges.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+use \Automattic\Jetpack\Status;
+
+/**
+ * The WP_Customize_Control core class is loaded only on customize_register.
+ *
+ * @param \WP_Customize_Manager $customize_manager Core customize manager.
+ */
+function register_css_nudge_control( \WP_Customize_Manager $customize_manager ) {
+	require_once __DIR__ . '/additional-css/class-css-nudge-customize-control.php';
+	require_once __DIR__ . '/additional-css/class-wpcom-css-customizer-nudge.php';
+
+	$domain = ( new Status() )->get_site_suffix();
+
+	if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+		require_once __DIR__ . '/additional-css/class-wpcom-additional-css-manager.php';
+		$manager = new WPCOM_Additional_CSS_Manager( $domain );
+	} else {
```

Comment:
Should we add a `jetpack_is_atomic_site` check here?

Reply:
It's not needed because we load the bootstrap.php script only for Atomic, and it's already guarded in masterbar.php

Reply:
It may be safer, to avoid issues if/when we eventually make this available to non-Atomic Jetpack sites.